postinstall.js

```diff
@@ -0,0 +1,5 @@
+const fs = require('fs');
```

Comment:
@herzog31 you can probably add this file to the `.hlxignore` list

Reply:
Thanks for the suggestion. Fix available in https://github.com/hlxsites/aem-boilerplate-commerce/pull/45.